dart-ts-mode.el

```diff
@@ -149,12 +149,11 @@ PARENT is always optional_formal_parameters."
       (+ (dart-ts-mode--parent-start parent) dart-ts-mode-indent-offset))))
 
 (defvar dart-ts-mode--keywords
-  '("async" "async*" "yield" "sync*"
-    "await" "get" "interface" "show"
-    "hide" "on" "class" "enum" "extends"
-    "in" "is" "new" "return"
-    "super" "this" "with" "if" "else" "when"
-    "try" "catch" "default" "switch")
+  '("async" "async*" "await" "catch" "class"
+    "default" "else" "enum" "extends" "get" "hide"
+    "if" "in" "interface" "is" "new" "on" "return"
+    "show" "super" "switch" "sync*" "this"
```

Comment:
Will "super" and "this" conflict with (super) and (this) below?

Reply:
They will be overrided, and they're in the same feature, we don't need to specify `:override t`.

Do you think we should remove "super" and "this" from `dart-ts-mode--keywords`?

I'd like to keep them since they're indeed keywords, and we changed their font-lock-face according to their actual work.

Reply:
We can keep them, I just worry about the conflict.